integrationtests/tests/fixtures/mod.rs

```diff
@@ -104,7 +105,7 @@ pub async fn fixtures(
     GatewayTest,
     Box<dyn LightningTest>,
 ) {
-    let base_port = BASE_PORT.fetch_add(num_peers * 2 + 1, Ordering::Relaxed);
+    let base_port = BASE_PORT.fetch_add(num_peers * 6 + 1, Ordering::Relaxed);
```

Comment:
If you make it just 10 or even 20, then it will avoid bumping it again in the future, and make it easier to remember which port is which as they will always end at the same digit.

Reply:
Good point, that'll be enough ports to last a while